frontend/__tests__/containers/ConfigStep/Board.test.tsx

```diff
@@ -84,13 +98,22 @@ describe('Board', () => {
   });
 
   it('should show board type when select board field value ', async () => {
-    setup();
+    const { getByRole, getByText } = setup();
 
-    fireEvent.mouseDown(screen.getByRole('button', { name: CONFIG_TITLE.BOARD }));
-    fireEvent.click(screen.getByText(BOARD_TYPES.CLASSIC_JIRA));
+    await userEvent.click(getByRole('button', { name: /board jira/i }));
 
     await waitFor(() => {
-      expect(screen.getByText(BOARD_TYPES.CLASSIC_JIRA)).toBeInTheDocument();
+      expect(getByRole('option', { name: /jira/i })).toBeInTheDocument();
```

Comment:
Avoid destructuring queries from `render` result, use `screen.getByRole` instead

See the detail in PR diff parts